index.js

```diff
@@ -1,291 +1,430 @@
+                                    let {torrent, title} = await app.publishTitle(body.folder)
+                                    if(prog.has(torrent.infoHash)){
+                                        prog.delete(torrent.infoHash)
+                                        prog.set(torrent.infoHash, torrent)
+                                    } else {
+                                        prog.set(torrent.infoHash, torrent)
+                                    }
+                                    if(req.mainReq){
+                                        res.data = [`<html><head><title>${torrent.title}</title></head><body><div><p>infohash: ${torrent.infoHash}</p><p>folder: ${title}</p></div></body></html>`]
+                                    } else {
+                                        res.data = [JSON.stringify({infohash: torrent.infoHash, title})]
+                                    }
+                                    res.statusCode = 200
+                                } else {
+                                    let {torrent, secret} = await app.publishAddress(body.folder, {address: body.address, secret: body.secret})
+                                    if(prog.has(torrent.address)){
+                                        prog.delete(torrent.address)
+                                        prog.set(torrent.address, torrent)
+                                    } else {
+                                        prog.set(torrent.address, torrent)
+                                    }
+                                    if(req.mainReq){
+                                        res.data = [`<html><head><title>${torrent.address}</title></head><body><div><p>address: ${torrent.address}</p><p>infohash: ${torrent.infoHash}</p><p>sequence: ${torrent.sequence}</p><p>signature: ${torrent.sig}</p><p>magnet: ${torrent.magnet}</p><p>secret: ${secret}</p></div></body></html>`]
+                                    } else {
+                                        res.data = [JSON.stringify({address: torrent.address, infohash: torrent.infoHash, sequence: torrent.sequence, magnet: torrent.magnet, signature: torrent.sig, secret})]
+                                    }
+                                    res.statusCode = 200
+                                }
+                            } else {
+                                if(req.mainReq){
+                                    res.data = [`<html><head><title>BT-Fetch</title></head><body><div><p>body is missing data</p></div></body></html>`]
+                                } else {
+                                    res.data = [JSON.stringify('body is missing data')]
+                                }
+                                res.statusCode = 400
+                            }
+                        }
+                        res.headers['Content-Type'] = req.mainRes
+                    } else {
+                        if(req.mainQuery.length === 64){
```

Comment:
this is when a user has stopped a previous torrent that was running, and then they want to restart that same torrent.

Reply:
Would it be possible to fold that functionality into doing a `GET` on the regular bittorrent URL with the torrent info instead?

Reply:
`GET` is used to load torrents of other user's, meaning `webtorrent.add()`. i will have to think about this and see how it can be done in the best way.